src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -88,6 +103,21 @@ public override void Initialize([AllowNull] ViewWithData data)
                 trackingAssignees.AddItem(pr.Assignee);
                 trackingAuthors.AddItem(pr.Author);
             }, () => { });
+
+            pullRequests.OriginalCompleted
+                .ObserveOn(RxApp.MainThreadScheduler)
+                .Subscribe(_ =>
+                {
+                    if (listSettings.SelectedAuthor != null)
+                    {
+                        SelectedAuthor = Authors.FirstOrDefault(x => x.Login == listSettings.SelectedAuthor);
+                    }
+
+                    if (listSettings.SelectedAssignee != null)
+                    {
+                        SelectedAssignee = Assignees.FirstOrDefault(x => x.Login == listSettings.SelectedAssignee);
+                    }
```

Comment:
If both are set, this causes the list to be refiltered twice, which is a potentially costly operation. It would be preferable if we manually called `UpdateFilter` here for these things, but the problem is going to be that the UI won't update unless we set `SelectedAuthor` and `SelectedAssignee`, and that brings us back to the same problem. So let's add a TODO here noting that this needs some rethinking.